### ISSUE TITLE
Loss value

### ISSUE THREAD

**Issue by viettmab** [rosinality/glow-pytorch#35]
I have a question about the loss value.

The loss value is the negative log likelihood (bits per dim) in paper, isn't it? When I trained the model, the loss value was much smaller than the best value in paper.

Thank you

**Comment by rosinality** [comment]
I remember it is due to train-test gap.

**Comment by mahaichuan** [comment]
Use $n_bits=8$.
